### PR TITLE
Adding connector code.

### PR DESCRIPTION
**Pull request by cs-amol-phatangade**

### CODE REVIEW

File: azure-storage-table/connector.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+from connectors.core.connector import Connector, get_logger, ConnectorError
+from .operations import operations, _check_health
+from connectors.core.utils import update_connnector_config
+
+logger = get_logger('azure-storage-table')
+
+CONFIG_SUPPORTS_TOKEN = True
+
+
+class MicrosoftSentinel(Connector):
```
Comment (cs-utkarsh-surana) [inline azure-storage-table/connector.py]: Please change the class name to AzureStorageTable

File: azure-storage-table/info.json [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+  "version": "1.0.0",
+  "description": "Azure Table storage is a service that stores non-relational structured data (also known as structured NoSQL data) in the cloud, providing a key/attribute store with a schemaless design. these connector helps you to create, update, delete, query on azure storage table",
+  "publisher": "Fortinet",
+  "cs_approved": true,
+  "cs_compatible": true,
+  "category": "IT Services",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "help_online": "",
+  "ingestion_supported": true,
```
Comment (cs-utkarsh-surana) [inline azure-storage-table/info.json]: Does this supports ingestion

File: azure-storage-table/info.json [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+          "title": "Table Name",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "table_name",
+          "description": "Table name to delete table."
+        }
+      ],
+      "output_schema": {},
```
Comment (cs-utkarsh-surana) [inline azure-storage-table/info.json]: Please add output schema if there is no output can we return a message for e.g. (deleted successfully) if the action execute successfully.

File: azure-storage-table/info.json [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+        "value": [
+          {
+            "TableName": ""
+          }
+        ]
+      },
+      "enabled": true
+    },
+    {
+      "title": "Insert Entity into Table",
```
Comment (cs-utkarsh-surana) [inline azure-storage-table/info.json]: change the title to Insert Entity **Into** Table

Similarly do it for rest of the titles

File: azure-storage-table/info.json [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+      "annotation": "query_table",
+      "parameters": [
+        {
+          "title": "Query Filter",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "query_filter",
+          "description": "Query expression."
```
Comment (cs-utkarsh-surana) [inline azure-storage-table/info.json]: Can we change the description to:
Specify the filter query based on which you want to retrieve tables from Azure Storage Table.

File: azure-storage-table/info.json [hunk 6]
```diff
@@ -0,0 +1,337 @@
...
+        },
+        {
+          "title": "Records Limit",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "records_limit",
+          "description": "Retrieve top n tables."
```
Comment (cs-utkarsh-surana) [inline azure-storage-table/info.json]: Can we change the description to:
Specify the maximum number of tables that this operation should return from Azure Storage Table.

File: azure-storage-table/info.json [hunk 7]
```diff
@@ -0,0 +1,337 @@
...
+          "description": "Unique identifier for an entity within a given partition."
+        },
+        {
+          "title": "Query Filter",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "query_filter",
+          "description": "Filter Entities query expression.\nInformation about Query expression structure can be found here: https://docs.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#constructing-filter-strings."
```
Comment (cs-utkarsh-surana) [inline azure-storage-table/info.json]: Update this description also.

File: azure-storage-table/info.json [hunk 8]
```diff
@@ -0,0 +1,337 @@
...
+          "title": "Entity Fields ",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "json",
+          "name": "entity_fields",
+          "description": "Entity fields in JSON format: { \"Key1\": Value1, \"Key2\": Value2}."
+        }
+      ],
+      "output_schema": {},
```
Comment (cs-utkarsh-surana) [inline azure-storage-table/info.json]: Add output schema, if not present return a successful message.

File: azure-storage-table/info.json [hunk 9]
```diff
@@ -0,0 +1,337 @@
...
+          "title": "Row Key",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "row_key",
+          "description": "Unique identifier for an entity within a given partition."
+        }
+      ],
+      "output_schema": {},
```
Comment (cs-utkarsh-surana) [inline azure-storage-table/info.json]: Add output schema, if not present return a successful message.

File: azure-storage-table/info.json [hunk 10]
```diff
@@ -0,0 +1,337 @@
...
+      "enabled": true
+    }
+  ]
+}
+
```
Comment (cs-utkarsh-surana) [inline azure-storage-table/info.json:343]: Please arrange all the actions in the CRUD sequence

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**